src/renderer/pages/Settings/Networks/Networks.tsx

```diff
@@ -150,27 +157,15 @@ export const Networks = () => {
     };
   };
 
-  const changeCustomNode = (network: ExtendedChain) => {
-    return (node?: RpcNode) => {
-      setNodeToEdit(node);
-      setNetwork(network);
-
-      toggleCustomRpc();
-    };
+  const closeAddCustomRpcModal = () => {
```

Comment:
This might be changed by `useModalClose` but not sure
At least it incapsulates a delay value and provides `closeModal` function
We use it from time to time, at least worth checking 😉

Reply:
This is something I wasn't sure about and I've asked earlier. The problem with managing the state of the modal in React (with `useModal` for instance), is that I need somehow to pass a callback to the effector model so that I close the modal e.g, when the new rpc node has been added/updated. So either I have:
- React state and I need to pass and `onClose` callback to the model
- Effector state and I need to play with some events in React to open/close the modal

At least that what I've been thinking about. Maybe there's something better, let me know.

Reply:
In "AddProxy" I use this:
```javascript
const [isModalOpen, closeModal] = useModalClose(!addProxyUtils.isNoneStep(step), addProxyModel.output.flowFinished);
```
But because I've got steps this is really handy, not sure about your situation

Reply:
Oh I see, sure, I guess I can make it work with it too. Let me check.

Reply:
done